crates/service/src/routes/request_handler.rs

```diff
@@ -54,5 +55,10 @@ pub async fn request_handler(
     let mut response = Response::new(body);
     response.extensions_mut().insert(attestation_input);
 
+    let graph_indexed = response.headers().get(GRAPH_INDEXED).cloned();
```

Comment:
This line needs to be moved above where `let mut response = Response::new(body);` aliases `response` away from the graph-node response.